test/e2e/kubectl.go

```diff
@@ -498,25 +498,29 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 			By("checking the result")
 			forEachReplicationController(c, ns, "app", "redis", validateReplicationControllerConfiguration)
 		})
-		It("should reuse nodePort when apply to an existing SVC", func() {
+		It("should reuse port when apply to an existing SVC", func() {
+			// This test doesn't do anything meaningful.  It was supposed to test that the nodeport
```

Comment:
These changes were already made on master, and should have been backported to 1.4 branch, but weren't.  This backports the changes to make the version skew test pass.